website/static/js/tests/paginator.test.js

```diff
@@ -125,22 +126,22 @@ describe('Paginator', () => {
 
         });
 
-        it('more than 7 pages, currentPage more than 5 and numbersOfPages - 5, two ellipses',
+        it('more than 41 pages, currentPage more than 20 and numbersOfPages - 20, two ellipses',
             () => {
```

Comment:
I don't think this comment is quite right -- This says that the number of pages is more than 41, but in the test it is equal to 41. Same with the current page -- the comment says it's more than 20 but it's equal to 20.